docs/src/index.md

```diff
@@ -90,14 +90,14 @@ hm2 = heatmap!(ax2, xP, zP, Pₙ, colorrange = P_lims, colormap = Reverse(:bamak
 Colorbar(fig[1, 2], hm1)
 Colorbar(fig[2, 2], hm2)
 
-record(fig, "buoyancy_front.mp4", 1:length(times)) do i
+record(fig, "buoyancy_front.gif", 1:length(times)) do i
     n[] = i
 end
 
 nothing #hide
 ```
 
-![buoyancy_front](buoyancy_front.mp4)
+![buoyancy_front](buoyancy_front.gif)
```

Comment:
Just note that file size is bigger and this will have an imprint of the github repo size

Reply:
I'll change this back, I don't think its possible to have videos in the readme so thought this might work to fix it, but doesn't seem to.